README.md

```diff
@@ -1,14 +1,23 @@
 # QuickSubmit Plugin
 
-- OJS QuickSubmit Plugin
-- Version: (see version.xml)
+[![Build Status](https://travis-ci.com/pkp/quickSubmit.svg?branch=stable-3_4_0)](https://travis-ci.com/pkp/quickSubmit)
+[![OJS compatibility](https://img.shields.io/badge/ojs-3.5-brightgreen)](https://github.com/pkp/ojs/tree/stable-3_5_0)
+[![GitHub release](https://img.shields.io/github/v/release/pkp/quickSubmit?include_prereleases&label=latest%20release)](https://github.com/pkp/quickSubmit/releases)
```

Comment:
Is there a way to find out the last release for this branch?

Reply:
That's right, you can replace by this:
https://img.shields.io/github/v/release/pkp/quickSubmit?include_prereleases&sort=semver&filter=*3.4.0*&display_name=release

Reply:
OK, I first now see your comment above. OK I will try to do that and we need to maybe then always use the version number in the release name?

Reply:
Feel free to drop, it's not a big deal for me :)